tests/main/interfaces-personal-files/task.yaml

```diff
@@ -0,0 +1,96 @@
+summary: Ensure that the personal-files interface works.
+
+details: |
+    The personal-files interface allows access specific personal files or directories.
+
+prepare: |
+    # shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB/snaps.sh"
+    install_local test-snapd-sh
+
+    # shellcheck source=tests/lib/files.sh
+    . "$TESTSLIB/files.sh"
+
+    # Fist layer of dirs and files
+    ensure_file_exists_backup_real /root/.testfile1
+    ensure_file_exists_backup_real /root/testfile1
+    ensure_dir_exists_backup_real /root/.testdir1
+    ensure_dir_exists_backup_real /root/testdir1
+
+    # Second layer of dirs and files
+    ensure_file_exists_backup_real /root/.testdir1/.testfilé2
+    ensure_file_exists_backup_real "/root/.testdir1/test file2"
+    ensure_dir_exists_backup_real /root/.testdir1/.testdir2
+
+    # Not accessible dirs and files
+    ensure_dir_exists_backup_real /tmp/.testdir1
+    ensure_file_exists_backup_real /tmp/.testfile1
+    ensure_dir_exists_backup_real /tmp/root/
+    ensure_file_exists_backup_real /tmp/root/testfile2
+    ensure_file_exists_backup_real /tmp/root/.testfile2
```

Comment:
Why are these entries in /tmp dirs not accessible? The default policy should allow these due to the private /tmp for the snap.... Perhaps /var/tmp would be a better test?